static/kubeapps-objs.yaml

```diff
@@ -881,467 +818,515 @@ spec:
     metadata:
       annotations: {}
       labels:
-        name: kubeless-ui
+        app: kubeapps-dashboard
+        name: kubeapps-dashboard-api
     spec:
       containers:
       - args: []
-        env: []
-        image: bitnami/kubeless-ui:latest
-        name: ui
+        env:
+        - name: MONOCULAR_HOME
+          value: /monocular
+        image: bitnami/monocular-api:v0.5.4
+        imagePullPolicy: IfNotPresent
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8081
+          initialDelaySeconds: 600
+          timeoutSeconds: 10
+        name: api
         ports:
-        - containerPort: 3000
-          name: ui
-          protocol: TCP
+        - containerPort: 8081
         readinessProbe:
           httpGet:
-            path: /
-            port: 3000
-        stdin: false
-        tty: false
-        volumeMounts: []
-      - args:
-        - proxy
-        - -p
-        - "8080"
-        env: []
-        image: kelseyhightower/kubectl:1.4.0
-        name: proxy
-        ports: []
+            path: /healthz
+            port: 8081
+          initialDelaySeconds: 0
+          timeoutSeconds: 5
+        resources:
+          limits:
+            cpu: 100m
+            memory: 128Mi
+          requests:
+            cpu: 100m
+            memory: 128Mi
         stdin: false
         tty: false
-        volumeMounts: []
+        volumeMounts:
+        - mountPath: /monocular
+          name: cache
+        - mountPath: /monocular/config
+          name: config
+      - env:
+        - name: TILLER_NAMESPACE
+          value: kube-system
+        - name: TILLER_HISTORY_MAX
+          value: "0"
+        image: gcr.io/kubernetes-helm/tiller:v2.7.0
```

Comment:
Shall we upgrade to 2.7.2? There is a security fix in 2.7.2: https://github.com/kubernetes/helm/releases/tag/v2.7.2

Reply:
@arapulido I'll make the change in kubeapps/manifest#21 - can you review this there please?